concordium-consensus/tests/scheduler/SchedulerTests/KonsensusV1/EpochTransition.hs

```diff
@@ -464,19 +463,23 @@ testEpochTransitionPaydayOnly accountConfigs = runTestBlockState @P7 $ do
             ^. cpCooldownParameters . cpUnifiedCooldown
 
 -- | Test an snapshot epoch transition.
-testEpochTransitionSnapshotOnly :: [AccountConfig 'AccountV3] -> Assertion
+testEpochTransitionSnapshotOnly ::
+    forall m.
+    (m ~ PersistentBlockStateMonad P7 (PersistentBlockStateContext P7) (BlobStoreT (PersistentBlockStateContext P7) (LoggerT IO))) =>
+    [AccountConfig 'AccountV3] ->
+    Assertion
```

Comment:
I think changing the signature of `computeBakerStakesAndCapital` should allow simplifying this back to the original signature (and elsewhere).

```suggestion
testEpochTransitionSnapshotOnly :: [AccountConfig 'AccountV3] -> Assertion
```